http/mocks/things.go

```diff
@@ -6,10 +6,15 @@ import (
 	"github.com/mainflux/mainflux"
 	"github.com/mainflux/mainflux/things"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 var _ mainflux.ThingsServiceClient = (*thingsClient)(nil)
 
+// ErrToken is used to simulate internal server error.
+const ErrToken = "unavailable"
```

Comment:
Rename this to something else, this is convention for `error` type.

Reply:
Irrelevant. If you find the more appropriate name, feel free to rename.

Reply:
It is not irrelevant, we should be consistent, even in mock. This name is confusing.

Reply:
I agree with @anovakovic01 I